addon/pods/components/frost-list/component.js

```diff
@@ -70,6 +69,14 @@ export default Ember.Component.extend({
     })
   }),
 
+  //  Optional attrs for sm vertical-collection
+  //  https://github.com/runspired/smoke-and-mirrors/blob/develop/addon/components/vertical-collection.js
+  idForFirstItem: null,
+  scrollPosition: 0,
+  key: '@identity',
+  defaultHeight: 60,
```

Comment:
I can't find anywhere in the spec where the height is specified, but I think this is correct